manifests/proxydhcp.pp

```diff
@@ -49,6 +49,12 @@
     failover       => $failover,
   }
 
+  exec { 'setfacl_dhcp':
+    command => 'setfacl -R -m u:theforeman:rwx /etc/dhcp /var/lib/dhcpd',
+    path => '/usr/bin',
+    onlyif => 'test -f /var/lib/dhcp && test -f /etc/dhcp',
```

Comment:
This isn't idempotent so I'd prefer 2 separate execs with a ```unless => getfacl```. It also doesn't handle the case where a user has extended attributes disabled.